common/saber_base.h

```diff
@@ -52,6 +52,7 @@ extern SaberBase* saberbases;
     DEFINE_EFFECT(UNJAM)			\
     DEFINE_EFFECT(PLI_ON)			\
     DEFINE_EFFECT(PLI_OFF)                      \
+    DEFINE_EFFECT(DESTRUCT)			\
```

Comment:
Shouldn't this be OFF_BLAST instead?

Reply:
Well the idea is to be an EFFECT that can be used as a blade animation.
Plus play an SFX_destruct sound. Sounds ends, boom.